### PR TITLE
doc feedback for client generation

### PR DESCRIPTION
**Pull request by gracewilcox**
Been reading through the docs and this PR has some minor feedback on things I've stumbled on.

### CODE REVIEW

File: website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx [hunk 1]
```diff
@@ -33,7 +33,7 @@ model InputModel {
 }
 
 @doc("Read my resource")
-op GetModel is ResourceRead<InputModel>;
+op Get is ResourceRead<InputModel>;
```
Comment (timotheeguerin) [inline website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx:36]:
```suggestion
op get is ResourceRead<InputModel>;
```

while there should change to follow naming convention

File: website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Getting started with `tsp-client`
+title: `tsp-client` commands
```
Comment (timotheeguerin) [inline website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md:2]:
```suggestion
title: `tsp-client`
```
only or `tsp-client usage` maybe?

Comment (catalinaperalta) [reply website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md]: +1 to `tsp-client usage`

File: website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx [hunk 3]
```diff
@@ -186,15 +186,17 @@ public final class InputOptions {
...
 }
 ```
 
 </TabItem>
 </Tabs>
 
-### Rename operations and parameters
+### Renaming operations and parameters
+
+Simillarly, you can rename operations like in the example below:
```
Comment (catalinaperalta) [inline website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx:199]: Nit:
```suggestion
Similarly, you can rename operations like in the example below:
```

File: website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx [hunk 4]
```diff
@@ -270,6 +272,8 @@ public final class PetStoreNamespaceClient {
 </TabItem>
 </Tabs>
 
+You cannot at this moment rename paramters in the client.tsp file.
```
Comment (catalinaperalta) [inline website/src/content/docs/docs/howtos/Generate client libraries/04renaming.mdx:275]: I'd add more to this and show an example of having the `@clientName` decorator over a parameter in an example operation. And say:
```suggestion
You cannot at this moment rename parameters in the client.tsp file. You will need to add the `@clientName` decorator over the parameter directly, example:

```

File: website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Getting started with `tsp-client`
+title: `tsp-client`
```
Comment (catalinaperalta) [inline website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md:2]:
```suggestion
title: `tsp-client usage`
```

File: website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md [hunk 6]
```diff
@@ -29,7 +29,7 @@ the `-o` or `--output-dir` option.
 
 ### init
 
-Initialize the client library directory using a tspconfig.yaml. When running this command pass in a path to a local or remote tspconfig.yaml with the `-c` or `--tsp-config` flag.
+Initialize the client library directory using a tspconfig.yaml. When running this command pass in a path to a local or the URL of a remote tspconfig.yaml with the `-c` or `--tsp-config` flag. If remote, The tspconfig.yaml must include the specific commit in the path. (See example)
```
Comment (catalinaperalta) [inline website/src/content/docs/docs/howtos/Generate with tsp-client/tsp_client.md:32]: Nit:
```suggestion
Initialize the client library directory using a tspconfig.yaml. When running this command pass in a path to a local or the URL of a remote tspconfig.yaml with the `-c` or `--tsp-config` flag. If remote, the tspconfig.yaml must include the specific commit in the path. (See example below)
```

**Review by catalinaperalta — COMMENTED** [review]
Thanks for contributing to the docs!!

**Review by timotheeguerin — APPROVED** [review]

**Comment by gracewilcox** [comment]
Thank you so much for getting this merged!! Really appreciate it!

**Pull request merged**